### PR TITLE
dev-db/etcd: remove etcd 0.4.9

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 1]
```diff
@@ -100,6 +86,7 @@ RDEPEND="${RDEPEND}
 	)"
 
 RDEPEND="${RDEPEND}
+	amd64? ( app-admin/etcd-wrapper )
```
Comment (dm0-) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: Was this dependent on the architecture before?

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 2]
```diff
@@ -9,7 +9,7 @@ HOMEPAGE="http://coreos.com"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 arm arm64 x86"
-IUSE="etcd_protocols_1 etcd_protocols_2 selinux"
```
Comment (dm0-) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: The arm64 profile still references `etcd_protocols_1`, which should be dropped.

File: coreos-base/coreos/coreos-0.0.1.ebuild (hunk 1 above)
Comment (dm0-) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: Actually, as of #2520, it looks like it is intended to be enabled on arm64.

Comment (glevand) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: Yes, we want etcd-wrapper on arm64.  We should get an arm64 quay.io image in the next etcd release.
See https://quay.io/repository/coreos/etcd?tab=tags.

Comment (crawford) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: Ah, I branched before #2520 was merged. I've updated it to unconditionally include etcd-wrapper.

**Pull request merged**